tests/test_geoprocessing.py

```diff
@@ -2589,6 +2589,22 @@ def test_convolve_2d_single_thread(self):
             n_pixels ** 2 * 9 - n_pixels * 4 * 3 + 4)
         numpy.testing.assert_allclose(numpy.sum(target_array), expected_result)
 
+    def test_convolve_2d_bad_path_bands(self):
+        """PGP.geoprocessing: test convolve 2d bad raster path bands."""
+        import pygeoprocessing

```

Comment:
Just flagging in case you are removing these imports in another PR or branch, then this new one might not be caught.